firebase-appdistribution/test-app/src/main/java/com/googletest/firebase/appdistribution/testapp/MainActivity.kt

```diff
@@ -63,20 +65,29 @@ class MainActivity : AppCompatActivity() {
         progressPercent = findViewById<TextView>(R.id.progress_percentage)
         progressBar = findViewById<ProgressBar>(R.id.progress_bar)
 
+        screenshotTriggerThread = HandlerThread("AppDistroFeedbackTrigger")
+        screenshotTriggerThread.start()
+        screenshotTrigger =
+            ScreenshotDetectionFeedbackTrigger(
+                this,
+                "Here's some terms and conditions",
```

Comment:
this should be from the `string` resource file, right?

Reply:
Yeah probably better to use a resource